content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst

```diff
@@ -0,0 +1,63 @@
+=================
+Add new equipment
+=================
+Using Odoo Maintenance, it is possible to track individual pieces of equipment, along with
+information about their maintenance requirements. To add a new piece of equipment, navigate to the
+:guilabel:`Maintenance` module, select :menuselection:`Equipments --> Machines & Tools --> Create`,
+and configure the equipment as follows:
```

Comment:
- italicize app name when not being included in a `menuselection` tag
- define _equipment_ and provide some context around it
```suggestion
Using Odoo *Maintenance*, it is possible to track individual pieces of equipment, along with
information about their maintenance requirements.

*Equipment* in this context refers to tools and machinery that's used on manufacturing or
engineering production lines, which can be owned by the company that uses the Odoo database, or a
third-party source such as a vendor in the case of equipment rentals.

To add a new piece of equipment, navigate to the :guilabel:`Maintenance` module, select
:menuselection:`Equipments --> Machines & Tools --> Create`, and configure the equipment as follows:
```